db/migrate/20241126084446_add_newcomer_reserved_spots_to_competitions.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddNewcomerReservedSpotsToCompetitions < ActiveRecord::Migration[7.2]
+  def change
+    add_column :Competitions, :newcomer_reserved_spots, :integer, default: 0, null: false
```

Comment:
You could use `null` here as a way of saying "I don't want to specify such a limit".
Although admittedly, `0` serves the same effect, so I have only a slight bias towards an actual `null` column

Reply:
The other PR will probably be merged first :wink:

Reply:
Now that other PR is merged, so you can go full steam ahead with `null` support here